cpp/rrtmgp/mo_gas_optics_rrtmgp.h

```diff
@@ -909,10 +906,6 @@ class GasOpticsRRTMGP : public OpticalProps {
     if (! this->is_initialized()) { stoprun("ERROR: spectral configuration not loaded"); }
     // Check for presence of key species in ty_gas_concs; return error if any key species are not present
     this->check_key_species_present(gas_desc);
-    // Check input data sizes and values
-    if (size(play,1) != ncol || size(play,2) != nlay  ) { stoprun("gas_optics(): array play has wrong size"); }
```

Comment:
Why remove these checks?

Reply:
Here, `play` may have come from a padded view, and so each row may have trailing garbage values after the last entry. However, `size(play,1)=ncol` would still pass, so I could add that back (but may be confusing as to why the 2nd dim isn't checked).